markdown/admin/packcore.html.md.erb

```diff
@@ -0,0 +1,51 @@
+---
+title: HAWQ packcore utility
+---
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+  http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+## <a id="core_file_definition"></a>Core file
+
+Core file is a disk file that records the image of a process' memory in case the process crashes or terminates abruptly. This image can be later used to debug the state of process at the time when it was terminated. This information can be very useful to debug the cause failure.
```

Comment:
A core file is a disk file that records the image of a process' memory in case the process crashes or terminates abruptly. The information in this image is useful for debugging the state of a process at the time when it was terminated.